pkg/operator/operator.go

```diff
@@ -51,7 +50,7 @@ func NewOpenShiftControllerManagerOperator(
 	targetImagePullSpec string,
 	operatorConfigInformer operatorinformersv1.OpenShiftControllerManagerInformer,
 	proxyInformer configinformerv1.ProxyInformer,
-	kubeInformersForOpenshiftControllerManager informers.SharedInformerFactory,
+	kubeInformers v1helpers.KubeInformersForNamespaces,
```

Comment:
We use this struct with the static resource sync controller and the resource sync controller, too.

Reply:
@adambkaplan - would it be possible to specify the repos(s) / packages(s) where the static resource sync controller and resource sync controller are located?  I'm guessing they are in library-go, but some quick scans are not proving fruitful to me.

I'd like to do some cross referencing as part of the review.

Thanks

Reply:
wait I may have found it later in the review ... https://github.com/openshift/cluster-openshift-controller-manager-operator/pull/216/commits/09eb355f0d0a7866c17b28f5a340936a7326eeb5#diff-0d623dfd885adb20f991bda4c2453aebd732ca6dbb4d1d4be6e79805c3b48de6R129

Reply:
yep, that is it